app/ingestion/jd_mapping.json

```diff
@@ -0,0 +1,43 @@
+{
+  "mappings": {
+    "job_description": {
+      "properties": {
+        "companyName": {
+          "type": "keyword",
+          "store": false,
+          "index": true
```

Comment:
by default, every field in ES is indexed... you dont need this setting explicitly in the mappings

Reply:
I like being explicit